<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>695</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QStackedWidget" name="stackW">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>9</y>
     <width>581</width>
     <height>681</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page1_2">
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>651</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_33">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string> РЛС</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </item>
      <item>
       <widget class="QTableWidget" name="IUStableRLS">
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_36">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Обстановка</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="IUStableObject">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>70</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_44">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Команды</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="IUStableCom">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>70</number>
        </attribute>
        <column>
         <property name="text">
          <string>Номер</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Тип</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Принята</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Обработана</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Доклад</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IUSpbInsertKom">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Сформировать команду</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_45">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Доклады</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="IUStableDoc">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="IUSpbClose">
          <property name="text">
           <string>Закрыть</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page2_2">
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>541</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_46">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string> РЛС</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_47">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Тип РЛС :</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_48">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Номер РЛС: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_49">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Обстановка</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_50">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Команды</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>451</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="IUSgraphicsLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
